app/controllers/drivers_controller.rb

```diff
@@ -23,11 +23,11 @@ def create
   end
 
   def edit
-    @driver = Driver.find(params[:id])
+    driver = Driver.find(params[:id])
   end
 
   def update
-    @driver = Driver.find(params[:id])
+    driver = Driver.find(params[:id])
```

Comment:
huh if you do this what are you updating on line:32?

Reply:
is `driver` overwriting the driver helper methods response? but does it setup the instance variable too?

Reply:
> huh if you do this what are you updating on line:32?

the driver still updates on the index even after the change

Reply:
do you even need the line though?
this isnt immediately apparent how this is even working, if this is doing some sorcery with the helper definition id rather us change it back.

or have a before action thats doing the `@driver = Driver.find(params[:id])`